zigpy/zcl/clusters/general.py

```diff
@@ -1278,6 +1286,444 @@ class PollControl(Cluster):
+        0x20: ("Door Lock Controller", [], [0x0101]),
+        0x30: ("Temperature Sensor", [0x0402], []),
+        0x31: ("Pressure Sensor", [0x0403], []),
+        0x32: ("Flow Sensor", [0x0404], []),
+        0x33: ("Indoor Environment Sensor", [], []),
+    }
+
+    def create_device(self, ieee, type=None, remoteCommissioning=False):
+        application = self.endpoint.device.application
+        if ieee in application.devices:
+            return application.devices[ieee]
+        if not remoteCommissioning and (
+            0x9997 not in self._attr_cache or self._attr_cache[0x9997] < time.time()
+        ):
+            LOGGER.debug("Not in permit joining mode and not in remoteCommissioning")
+            return
+        LOGGER.debug("Device %s not found, create it", ieee)
+        dev = application.add_device(ieee, 32766)
+        dev.initializing = True
+        dev.status = zigpy.device.Status.ENDPOINTS_INIT
+        dev._skip_configuration = True
+        ep = dev.add_endpoint(1)
+        ep.status = zigpy.endpoint.Status.ZDO_INIT
+        ep.profile_id = zigpy.profiles.zha.PROFILE_ID
+        ep.device_type = zigpy.profiles.zha.DeviceType.GREEN_POWER
+        ep.add_input_cluster(zigpy.zcl.clusters.general.Basic.cluster_id)
+        ep.in_clusters[zigpy.zcl.clusters.general.Basic.cluster_id]._update_attribute(
+            0x0004, "GreenPower"
+        )
+        in_clusters = None
+        out_clusters = None
+        if type is not None and type in GreenPowerProxy.device:
+            name, in_clusters, out_clusters = GreenPowerProxy.device[type]
+            ep.in_clusters[
+                zigpy.zcl.clusters.general.Basic.cluster_id
+            ]._update_attribute(0x0005, name)
+            for cluster_id in in_clusters:
+                LOGGER.debug("Add input cluster id %s on device %s", cluster_id, ieee)
+                ep.add_input_cluster(cluster_id)
+            for cluster_id in out_clusters:
+                LOGGER.debug("Add output cluster id %s on device %s", cluster_id, ieee)
+                ep.add_output_cluster(cluster_id)
+        else:
+            ep.in_clusters[
+                zigpy.zcl.clusters.general.Basic.cluster_id
+            ]._update_attribute(0x0005, "GreenPowerDevice")
+        ep = dev.add_endpoint(zigpy.zcl.clusters.general.GreenPowerProxy.endpoint_id)
+        ep.status = zigpy.endpoint.Status.ZDO_INIT
+        ep.profile_id = zigpy.profiles.zha.PROFILE_ID
+        ep.device_type = zigpy.profiles.zha.DeviceType.GREEN_POWER
+        ep.add_input_cluster(zigpy.zcl.clusters.general.GreenPowerProxy.cluster_id)
+        application.device_initialized(dev)
+        return dev
+
+    def handle_notification(
+        self, ieee, header, counter, command_id, payload, payload_length, mic
+    ):
+        application = self.endpoint.device.application
+        if ieee not in application.devices:
+            return
+        if command_id not in GreenPowerProxy.command:
+            return
```

Comment:
this might be a good opportunity to log something here, along the lines "unhandled command_id`